### DISCUSSION TITLE
Can't find plot_rf()

### DISCUSSION THREAD

**Discussion by clawrie** [boschresearch/pylife#65] [General]
I'm trying to work through the cookbook without using Jupyter and can't work out where this import comes from?
`from helper_functions import plot_rf`
Can anyone help?

**Comment by johannes-mueller** [comment]
Hi @clawrie 

Thanks for reaching out.

It's in [`/demos/helper_functions.py`](https://github.com/boschresearch/pylife/blob/develop/demos/helper_functions.py)

@DKreuter we should think of a way to better handle this.
